scripts/safe_template_linter.py

```diff
@@ -2139,6 +2140,28 @@ def _check_javascript_context(self, javascript_code, start_offset, results):
         self.javascript_linter.check_javascript_file_is_safe(javascript_code, javascript_results)
         self._shift_and_add_violations(javascript_results, start_offset, results)
 
+    def _check_mako_python_blocks(self, mako_template, has_page_default, results):
+        """
+        Searches for Mako python blocks and checks if they contain
+        violations.
+
+        Arguments:
+            mako_template: The contents of the Mako template.
+            has_page_default: True if the page is marked as default, False
+                otherwise.
+            results: A list of results into which violations will be added.
+
+        """
+        python_block_regex = re.compile('<%\s(?P<code>.*?)%>', re.DOTALL)
```

Comment:
Reviewers: I voted on a simple regex, rather than trying to make sure this didn't appear in strings.